docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlOneOfTest.scala

```diff
@@ -4,16 +4,28 @@ import io.circe.generic.auto._
 import org.scalatest.funsuite.AnyFunSuite
 import org.scalatest.matchers.should.Matchers
 import sttp.model.StatusCode
+import sttp.model.StatusCode.NoContent
 import sttp.tapir.docs.openapi.VerifyYamlOneOfTest._
 import sttp.tapir.generic.auto._
 import sttp.tapir.json.circe.jsonBody
 import sttp.tapir.openapi.Info
 import sttp.tapir.openapi.circe.yaml._
 import sttp.tapir.tests.MultipleMediaTypes
-import sttp.tapir.{Codec, CodecFormat, Schema, SchemaType, endpoint, header, plainBody, statusCode, oneOfDefaultMapping, oneOfMapping}
+import sttp.tapir.{Codec, CodecFormat, EndpointIO, Schema, SchemaType, emptyOutput, endpoint, header, oneOfDefaultMapping, oneOfMapping, plainBody, statusCode, statusDefaultMapping, statusMapping}
 
 class VerifyYamlOneOfTest extends AnyFunSuite with Matchers {
 
+  test("should support description for status code mappings with empty output") {
+    val expectedYaml = load("oneOf/expected_status_codes_with_empty_output.yaml")
+
+    val e = endpoint.errorOut(sttp.tapir.oneOf(oneOfMapping(StatusCode.Forbidden, EndpointIO.Empty(Codec.idPlain(), EndpointIO.Info.empty).description("forbidden"))))
```

Comment:
maybe you can use `emptyOutput` instead of `EndpointIO.Empty(Codec.idPlain(), EndpointIO.Info.empty)`?

Reply:
I already tried but `endpointOutput` is defined as 
`  val emptyOutput: EndpointOutput[Unit] = EndpointIO.Empty(Codec.idPlain(), EndpointIO.Info.empty)`
and it's type doesn't allow to specify description as `EndpointIO` does, can I change `emptyOutput's` type to `EndpointIO` ?

Reply:
Ah ... yes, I think returning a more specific type (I think it will need to be even more specific than EndpointIO) is fine

Reply:
all the other methods return specific return types